dbildungs-iam-secrets-secret-dev.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: onepassword.com/v1
```

Comment:
I think it would be nicer to have these files in an own folder, in the future further manifest files may also be needed.